katalog/nginx/bases/controller/DaemonSet-ingress-nginx-controller.yml

```diff
@@ -37,27 +55,28 @@ spec:
                   - /wait-shutdown
           args:
             - /nginx-ingress-controller
-            - --configmap=$(POD_NAMESPACE)/nginx-configuration
             - --publish-service=$(POD_NAMESPACE)/ingress-nginx
-            - --annotations-prefix=nginx.ingress.kubernetes.io
```

Comment:
is this flag not needed anymore?

Reply:
It's not mentioned in the chart, but from [here](https://github.com/kubernetes/ingress-nginx/blob/main/docs/user-guide/cli-arguments.md) I can see it's the default value.